src/utils/api/functionMapper.js

```diff
@@ -1,22 +1,28 @@
 import { tokenMap } from '../../constants/tokens';
 import * as btcAccount from './btc/account';
-import * as btcTransactions from './btc/transactions';
-import * as btcService from './btc/service';
 import * as btcNetwork from './btc/network';
+import * as btcService from './btc/service';
+import * as btcTransactions from './btc/transactions';
+import * as lskAccount from './account';
 import * as lskNetwork from './lsk/network';
+import * as lskService from './liskService';
```

Comment:
IMO it should be moved to `lsk/` folder

```suggestion
import * as lskService from './lsk/service';
```

Reply:
you are right I just moved to the correct folder